### ISSUE TITLE
2.28.1 - Annotations tab not showing all annotations

### ISSUE THREAD

**Issue by krchristie** [geneontology/paint#44]
Hi Suzi,

I loaded PTHR38687 into 2.28.1 and am seeing something weird. The Annotation Matrix shows 4 annotations total (1 MF, 2 CC, and 1 BP). However, the Annotations tab only shows 2 annotations (1 MF and 1 CC). The previous version 2.28 is fine in this regard. For both versions, I loaded this family as New. May or may not be relevant, but I notice that v2.28.1 is only showing one annotation each from each of two references which should each have two annotations.

Screenshots of each attached. I selected the BP tab since it's more obvious to see that there aren't any annotations with that background color in the Annotations tab.

thanks,

Karen

![20161129-2 28 1-missingannots](https://cloud.githubusercontent.com/assets/10533218/20728755/00b632da-b634-11e6-8e00-429692c58cf1.jpg)

![20161129-2 28-comp](https://cloud.githubusercontent.com/assets/10533218/20728760/0987b7a8-b634-11e6-9fbf-098d796bf396.jpg)

**Comment by krchristie** [comment]
Actually this isn't specific to 2.28.1, and doesn't seem to be consistent.

Later on 11/29/16, I loaded a family (PTHR38603) into 2.28 and it was missing some annotations in the Annotations tab that were shown in the Annotation Matrix.
![20161129-2 28-missingbp](https://cloud.githubusercontent.com/assets/10533218/20759915/43a416a8-b6d3-11e6-8080-ef9c4cad520b.jpg)

Then on 11/30/16, I loaded that same family into both 2.28 and 2.28.1 and both were fine. 
![20161130-2 28-pthr38603](https://cloud.githubusercontent.com/assets/10533218/20759945/5875baf0-b6d3-11e6-9558-c9c090bec8a9.jpg)

![21061130-2 28 1-pthr38603](https://cloud.githubusercontent.com/assets/10533218/20759963/63ec7450-b6d3-11e6-8047-3b203a07d749.jpg)
